lib/wallet.js

```diff
@@ -0,0 +1,96 @@
+import React, { useCallback, useContext, useMemo } from 'react'
+import env from './environment'
```

Comment:
Minor style nitpick: we generally order imports to make it clearer which imports are "controlled" by us:

1. React-related (`react`, `react-dom`, `styled-components`, etc)
1. External dependencies we don't control
1. External dependencies we do control
1. Internal modules

So to apply in this case, we would reorder to:

1. React
1. `ethers`
1. `use-wallet`
1. Internal modules